README.md

```diff
@@ -1,14 +1,11 @@
 # angular-material-tools [![Build Status](https://travis-ci.org/angular/material-tools.svg?branch=master)](https://travis-ci.org/angular/material-tools)
 
-[Angular-Material](http://github.com/angular/material) deploys standard builds to NPM and Bower using the [Bower-Material](http://github.com/angular/bower-material) deployment repository.
+NodeJS/TypeScript/CLI tool that generates custom [Angular-Material v1.x](http://github.com/angular/material) builds, consisting of:
```

Comment:
I don't like that slashes here.

I'd just say
- `Tool to generate custom [Angular-Material v1.x](http://github.com/angular/material) builds`

@ThomasBurleson Any better suggestions?